### PR TITLE
OHRI-1214 Use previous value functionality should be configurable for specific fields and/ or forms

### PR DESCRIPTION
**Pull request by kajambiya**

### CODE REVIEW

File: src/api/types.ts [hunk 1]
```diff
@@ -68,6 +68,9 @@ export interface OHRIFormSchema {
   inlineRendering?: 'single-line' | 'multiline' | 'automatic';
   markdown?: any;
   postSubmissionActions?: Array<string>;
+  formOptions?: {
+    usePreviousValueDisabled: boolean;
+  };
```
Comment (samuelmale) [inline src/api/types.ts:73]: We may need to move other options like:
- inlineRendering
- readonly
- defaultPage
-  allowUnspecifiedAll
- postSubmissionActions

Under `formOptions`. @kajambiya Can you log a ticket for this?

**Review by samuelmale — COMMENTED** [review]
Does this also cater for the Form level config?

File: src/api/types.ts (hunk 1 above)
Comment (kajambiya) [reply src/api/types.ts:73]: Sure. Let me do that

**Comment by kajambiya** [comment]
> Does this also cater for the Form level config?

It does.
The one under `OHRIFormQuestionOptions` is for field level and the one under `OHRIFormSchema` is global

**Review by samuelmale — APPROVED** [review]

**Pull request merged**